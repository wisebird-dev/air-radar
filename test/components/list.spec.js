import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import List from '../../pages/components/list';

test('List renders correctly', async () => {
	const tree = await renderer.create(<List/>).toJSON();

	expect(tree).toMatchSnapshot();
});
