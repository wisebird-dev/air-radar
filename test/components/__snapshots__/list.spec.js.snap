// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List renders correctly 1`] = `
.c0 {
  position: relative;
  list-style: none;
  margin-left: 0;
  padding-left: 1.2em;
  padding-bottom: 12px;
}

.c0 li:before {
  content: "â€º";
  position: absolute;
  left: 0;
}

<ul
  className="c0"
/>
`;
